src/packs.rs

```diff
@@ -86,6 +87,9 @@ pub fn check(
         OutputFormat::CSV => {
             csv::write_csv(&result, std::io::stdout())?;
         }
+        OutputFormat::JSON => {
+            json::write_json(&result, std::io::stdout())?;
```

Comment:
We should exit with a non-zero status when there are violations:

```suggestion
            json::write_json(&result, std::io::stdout())?;
            if result.has_violations() {
                std::process::exit(1);
            }
```

I wonder if we should use a different code versus a runtime error? This way, callers don't have to differentiate "runtime error as plain text" versus "violations presented as JSON".